modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomLayouts.macro

```diff
@@ -27,11 +27,11 @@ definition {
 			ClickNoError(
 				key_titleFieldName = ${selectEntry},
 				locator1 = "CreateObject#SELECT_ENTRY_IN_RELATIONSHIP_TAB");
+
+			Refresh();
```

Comment:
I guess that when the testcase into on SelectFrame, it needs to get out after. So, I put a `Refresh();` to reload the page and the function see the kebab.

Reply:
cc: @timpak